braviarc/braviarc.py

```diff
@@ -146,6 +146,23 @@ def send_command(self, command):
         """Sends a command to the TV."""
         self.send_req_ircc(self.get_command_code(command))
 
+    def get_source(self, source):
+        """Returns list of Sources"""
+        original_content_list = []
+        stIdx = 0
```

Comment:
var name does not fit python coding standards (https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)